### PR TITLE
flight: avoid getting uavos unnecessarily

### LINKED ISSUE #471: Request for comment: change code pattern
Very frequently with UAVOs, we do something like:

StabilizationDesiredGet(&stabDesired);
stabDesired.Throttle=...
stabDesired.Whatever=...
StabilizationDesiredSet(&stabDesired);

I think if we intend to set all fields, we should default or bracket initialize things instead of the initial get.  This way we're not tainted by past states, we don't contend the giant uavo lock, and we don't copy data as much.

Thoughts?

### PR DESCRIPTION
**Pull request by mlyle**
Largely fixes #471.

Avoids copying lots of data and removes some pressure on UAVO lock.

### CODE REVIEW

**Comment by mlyle** [comment]
Flight tested, including autotune, on Sparky2.

Flight tested, including autotune, on Naze32.

File: flight/Modules/Attitude/attitude.c [hunk 1]
```diff
@@ -1410,7 +1406,6 @@ static void updateNedAccel()
 	accel_ned[2] += GRAVITY;
 	
 	NedAccelData accelData;
-	NedAccelGet(&accelData);
```
Comment (tracernz) [inline flight/Modules/Attitude/attitude.c:1413]: This one is set by GroundPathFollower as well, https://github.com/d-ronin/dRonin/blob/080150a5b271fcfb3fe36aa1e20f92fc0d956aca/flight/Modules/GroundPathFollower/groundpathfollower.c#L484

File: flight/Modules/ManualControl/failsafe_control.c [hunk 2]
```diff
@@ -78,7 +78,6 @@ int32_t failsafe_control_select(bool reset_controller)
 	SystemSettingsAirframeTypeGet(&airframe_type);
 
 	StabilizationDesiredData stabilization_desired;
-	StabilizationDesiredGet(&stabilization_desired);
```
Comment (tracernz) [inline flight/Modules/ManualControl/failsafe_control.c:81]: This one is set in various places.

File: flight/Modules/Attitude/attitude.c (hunk 1 above)
Comment (mlyle) [inline flight/Modules/Attitude/attitude.c:1413]: Yes, but we fully set all the values.

Comment (tracernz) [inline flight/Modules/Attitude/attitude.c:1413]: Yeah, I'm thinking of the UAVO fields changing in future though, it'll be quite fragile.

**Comment by tracernz** [comment]
Discussed on IRC. Setting UAVOs in multiple places like this is problematic in general and StabDesired is already in plan for future work. As long as we are writing all the fields what is done here will be okay/better than it was before.

**Pull request merged**